src/App.js

```diff
@@ -1,11 +1,23 @@
 import React from 'react';
 import './App.scss';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
+
+import Login from 'views/Auth/Login'
+import Register from 'views/Auth/Register'
+import ForgetPassword from 'views/Auth/ForgetPassword'
+import Welcome from 'views/Welcome'
 
 function App() {
   return (
-    <div className="App">
-    <p>hello</p>
-    </div>
+    <Router>
+        <Switch>
+          <Route path="/" component={Welcome} exact/>
+          <Route path="/login" component={Login}/>
+          <Route path="/register" component={Register}/>
+          <Route path="/forgetpassword" component={ForgetPassword}/>
```

Comment:
you should add `auth` to the routes so the appear like this auth/login, auth/register, and auth/forgotPassword

Reply:
also do not forget to change to forgotPassword where necessary